build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -488,22 +489,57 @@ export class FluidPackageCheck {
     private static checkLintScripts(pkg: Package, fix: boolean) {
         let fixed = false;
         if (pkg.getScript("build")) {
-            const hasPrettier = pkg.getScript("prettier");
-            const lintChildren = hasPrettier ? ["prettier", "eslint"] : ["eslint"];
-            if (this.checkChildrenScripts(pkg, "lint", lintChildren, false, fix)) {
-                fixed = true;
+            // hasPrettier commented out to discard build warnings from "lint" & "lint:fix" scripts
+            // const hasPrettier = pkg.getScript("prettier");
+
+            // npm run lint
+            const lintScript = pkg.getScript("lint"); // acess lint-script
+            const lintPackages = lintScript?.match(/run\s*([^&]*)\s*/g); // regex to create a list of ["run foo", "run bar"]
```

Comment:
Nit: We repeat this same regex in both checks. Might be nice to extract this as a constant and document the expression contents to explain what we're trying to capture with it.

I always recommend documenting regex expressions both to enhance legibility, and to explain the *intention* in addition to the implementation.